lib/api_headers.js

```diff
@@ -11,8 +11,8 @@
  * @param {Response} res Response object
  * @param {NextFunction} next Override the next() function, next() function causes next middleware to be executed
  */
-export default function api_headers (req, res, next) {
-  let origin = [req.headers.origin].find((o) => o);
+export default function api_headers(req, res, next) {
+  let origin = [req.headers.origin].find(o => o);
```

Comment:
looks like Linting changes? Is this in alignment with the rules?

Reply:
It's in alignment with the rules that were changed with this PR yes. Mind you, a lot of the rule removals are ones that either are implicitly set via the configs we use and those that are made redundant and conflict with Prettier.